heron/spi/tests/java/com/twitter/heron/spi/utils/SchedulerUtilsTest.java

```diff
@@ -168,4 +175,63 @@ public void testSchedulerCommandArgs() throws Exception {
     Assert.assertArrayEquals(expectedArgs, SchedulerUtils.schedulerCommandArgs(
         Mockito.mock(Config.class), Mockito.mock(Config.class), freePorts));
   }
+
+  @Test
+  public void updatesPackingPlanWithUniformSizedContainers() {
+    SchedulerStateManagerAdaptor adaptor = Mockito.mock(SchedulerStateManagerAdaptor.class);
+    Mockito.when(adaptor.deletePackingPlan("topology")).thenReturn(true);
+    Mockito.when(adaptor.setPackingPlan(Mockito.any(PackingPlans.PackingPlan.class),
+        Mockito.eq("topology"))).thenReturn(true);
+    Config runtime = Mockito.mock(Config.class);
+    Mockito.when(runtime.get(Keys.schedulerStateManagerAdaptor())).thenReturn(adaptor);
+
+    PackingPlan packing = createPackingPlan();
+    SchedulerUtils.persistUpdatedPackingPlan("topology", packing, runtime);
+    InOrder inOrder = Mockito.inOrder(adaptor);
+    inOrder.verify(adaptor).deletePackingPlan("topology");
+    inOrder.verify(adaptor).setPackingPlan(Mockito.any(PackingPlans.PackingPlan.class),
+        Mockito.eq("topology"));
+  }
+
+  @Test
+  public void updatesPackingPlanInStateManager() {
```

Comment:
+1

Having some notion of the test method in the name (when possible) helps to clarify the scope of what the test if intended to verify. I find this useful for tests of limited scope, which I guess most should be ideally.